src/globals/resources/queue.resource.ts

```
@@ -1,30 +1,96 @@
+
+        return queue.run(task);
       },
     };
   },
   dispose: async (_value, _, _deps, context) => {
+    context.disposed = true;
+    context.cleanupTimers.forEach((timer) => {
+      clearTimeout(timer);
+    });
+    context.cleanupTimers.clear();
     context.map.forEach((queue) => queue.dispose());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)